### ISSUE TITLE
[Design] Create Final Wireframes for Adding Physical Location to Pre-Check-in & Check-in Appointment Display Screens

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#42723]
## Background
Wireframe explorations [ticket](https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/42030)

## Wireframe
Start with this version and remove the V2 fields (type of care and doctor name)
![image.png](https://images.zenhubusercontent.com/60db6a817900f619905e68b1/0149754f-e1f9-4b44-8cd7-9b39f21b2700)

## User Story
We need final wireframes for adding physical location field to displays of appointment information in **PRE-CHECK-IN AND CHECK-IN** applications

## Tasks
- [ ] Finalize detailed wireframes and callouts 
- [ ] Decide: Should a content and accessibility review be part of this ticket or separate tickets due to scope?

## Acceptance Criteria
- [ ] UI review meeting with product/UX team for feature capabilities
- [ ] UI review meeting with engineering for layout and callouts (can be the same meeting as above)
- [ ] Wireframe available on Sketch Cloud
- [ ] If the wireframes applies to an error state or text message, then update the [GitHub source of truth](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/design/text-and-error-messages.md) documentation.

**Comment by benbrasso-agile6** [comment]
Historical conversation in [42030](https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/42030)

**Comment by ytsaoca** [comment]
Add the location label and make the layout of CI and PCI consistent for current stage. 
CI confirmation page with location: https://www.sketch.com/s/e79a827e-42cf-4a82-b554-874c75b5c70e/a/v8GPamb

PCI confirmation page with location: https://www.sketch.com/s/5331b114-280d-4ff5-8d36-ec49b1696b9e/a/zxa1Jyr

I also explore other possibility to reorganize the information architecture of the confirmation page and would like to have content research in the future. 

Discovery: If there are more appointment information to show, what if keep the confirmation page concise with only next steps (blue alert and way to update personal information/NoK/Emergency) and have a link to the appointment information?

https://www.sketch.com/s/5331b114-280d-4ff5-8d36-ec49b1696b9e/a/Jn3ZM4b
https://www.sketch.com/s/5331b114-280d-4ff5-8d36-ec49b1696b9e/a/Jn34R8e

**Comment by benbrasso-agile6** [comment]
I just did a quick look @brianseek  @loripusey 

Why isn't facility field present on check-in confirmation page? 

Feels like Location should be on the Your Appointments page for check-in. Need to update Abstract. 

Was there a decision NOT to include location on the "Answer pre-check-in questions" intro page in pre-check-in? 

If we don't know the answers to those questions, we may want to wait for Kristen and Ya-ching to return. Not sure if Nira is close to this work or not.

**Comment by loripusey** [comment]
Can we just say that anywhere we display appointment information for in-person appointments we should include the Location? The pattern is the same for how we display the appointment info no matter what page it's on. Do we really need to halt forward progress because we don't have a wireframe for every page?

**Comment by niradatta1** [comment]
@loripusey @benbrasso-agile6 I think we're all on the same page, in that the location field should show up anywhere we display appointment info. I'm noting that we need to update the wireframes in parallel, so it's not a blocker to implementation and @brianseek can still proceed.

**Comment by loripusey** [comment]
I'll create a new ticket for updating the wireframes, I don't think a note will be enough. Thanks.

**Comment by loripusey** [comment]
@ytsaoca did this ticket ever go through validation?
cc: @zach-park @kristenmcconnell
